dask/Dask-Collections.py

```diff
@@ -215,17 +233,22 @@ def special_load_function(x):
 # In[ ]:
 
 
-bag.from_sequence(range(0,1000)).map(lambda x: (x, x)).foldby(lambda x, y: x + y, lambda x, y: x + y)
```

Comment:
@holdenk  technically pep8 is correct but pretty illegible after this edit. I'm going to undo this and just fit it within 80 char